src/contractTest/resources/application.yaml

```diff
@@ -296,8 +296,6 @@ document:
     citizen:
       prl_citizen_upload_template: PRL-CA-GOR-ENG-Draft-An-Order.docx
       prl_citizen_upload_filename: Self_created.pdf
-      respondent_c7_template: FL-PRL-GOR-ENG-Citizen-Uploaded-Statement.docx
-      respondent_c7_file_name: Citizen_Uploaded_Statement.pdf
```

Comment:
Dont we need this reference here?

Reply:
I see the build is green, but I feel this will break the Contract test.

Reply:
We dont need this, removed from other yaml files now